kimchi/src/circuits/polynomials/lookup.rs

```diff
@@ -142,29 +139,6 @@ use CurrOrNext::*;
 /// Number of constraints produced by the argument.
 pub const CONSTRAINTS: u32 = 7;
 
-// TODO: Update for multiple tables
-fn single_lookup<F: FftField>(s: &SingleLookup<F>) -> E<F> {
-    // Combine the linear combination.
-    s.value
-        .iter()
-        .map(|(c, pos)| {
-            E::literal(*c)
-                * E::Cell(Variable {
-                    col: Column::Witness(pos.column),
-                    row: pos.row,
-                })
-        })
-        .fold(E::zero(), |acc, e| acc + e)
-}
-
-fn joint_lookup<F: FftField>(j: &JointLookup<F>) -> E<F> {
-    j.entry
-        .iter()
-        .enumerate()
-        .map(|(i, s)| E::constant(ConstantExpr::JointCombiner.pow(i as u64)) * single_lookup(s))
```

Comment:
so if I understand correctly this is replaced by the logic in `combine_table_entry`

Reply:
Exactly, yes